src/hypothesis/strategies.py

```diff
@@ -945,11 +955,26 @@ def builds(target, *args, **kwargs):
     the target.
 
     """
+    if target_and_args:
+        target, args = target_and_args[0], target_and_args[1:]
+        if not callable(target):
+            raise InvalidArgument('The first positional argument to builds() must be a callable'
+                                  'target to construct.')
+    elif 'target' in kwargs:
```

Comment:
elif 'target' in kwargs and callable(kwargs['target']):

Handles the case where someone passed `target=some_strategy` but did not supply any positional arguments - we want that to be the generic "no callable supplied" error, and weren't checking.

Reply:
Good point, I'll also add a test that would have failed w/o this fix.